impress-org/givewp
# Fix: terms and conditions + consent field design updates

## Description

This PR updates the ```<ControlForPopover/>``` components, edit icon as well as replaces the Terms and Condition blocks original Popover control with this up to date version to improve consistency across the app. 

The Consent template has a styling change to properly align the ```Show Terms``` button on the frontend/design mode. It has also been updated to accommodate the differing required field labels and designs between the Consent and the Terms and Conditions block.
 
The width of the Terms and Conditions text editor modal has also been greatly reduced from 45rem to 25rem to match our other Popovers.

To help facilitate these changes I moved all the inline styles into a separate stylesheet on each sub directory.

Additionally the OfflineDonation Setting uses this same Popover control component. It previously was marked as "Edit" to open the Popover - this has been removed so that it uses the same icon as our others.

https://app.asana.com/0/1203547864809090/1205502083833591

## Affects
- Terms and Conditions block
- Consent field

## Visuals

https://github.com/impress-org/givewp/assets/75056371/5b6d853e-f99b-467c-9cf6-c4c5566a1bfe

## Testing Instructions
- Add the terms and conditions block in the new form builder.
- Set settings to customize.
- Set display type to Show in modal.
- On agreement text - use the pen to toggle the modal.
- View design mode and frontend page.
- Verify Link or button is aligned with the rest of the text.
- On the FormSettings - Select Donation Confirmation.
- Verify new icon displays for their popovers.
- Repeat last step for offline donation popover as well.

## Pre-review Checklist

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205226775556234
